src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/AutoDrier.java

```diff
@@ -0,0 +1,134 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.machines;
+
+import me.mrCookieSlime.CSCoreLibPlugin.general.Inventory.InvUtils;
+import me.mrCookieSlime.CSCoreLibPlugin.general.Inventory.Item.CustomItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems.AContainer;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems.MachineHelper;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems.MachineRecipe;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
+import me.mrCookieSlime.Slimefun.api.energy.ChargableBlock;
+import org.bukkit.Material;
+import org.bukkit.Tag;
+import org.bukkit.block.Block;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.Damageable;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AutoDrier extends AContainer {
+
+    public AutoDrier(Category category, ItemStack item, String id, RecipeType recipeType, ItemStack[] recipe) {
+        super(category, item, id, recipeType, recipe);
+    }
+
+    @Override
+    public String getInventoryTitle() {
+        return "&eAuto Drier";
+    }
+
+    @Override
+    public ItemStack getProgressBar() {
+        return new ItemStack(Material.CAMPFIRE);

```

Comment:
Look at the code below, the progress bar needs to be a damagable item.
Campfires do not have a damage bar afaik.